samples/net/eth_native_posix/README.rst

```diff
@@ -28,15 +28,43 @@ below.
    :compact:
 
 Normally one needs extra privileges to create and configure the TAP device in
-the host system. Use sudo to execute the Zephyr process with admin privileges,
-like this:
+the host system. If the user has set the
+:option:`CONFIG_ETH_NATIVE_POSIX_STARTUP_AUTOMATIC` option (this is disabled
+by default), then the user needs to use ``sudo`` to execute the Zephyr process
+with admin privileges, like this:
 
 .. code-block:: console
 
     sudo --preserve-env=ZEPHYR_BASE make run
 
 If the ``sudo --preserve-env=ZEPHYR_BASE`` gives an error,
-just use ``sudo --preserve-env`` instead:
+just use ``sudo --preserve-env`` instead.
+
+If the :option:`CONFIG_ETH_NATIVE_POSIX_STARTUP_AUTOMATIC` option
+is not enabled (this is the default), then the user should
+execute the ``net-setup.sh`` script from Zephyr `net-tools`_ repository.
+The script should be run before executing the Zephyr process. The script
+will create the zeth interface and set up IP addresses and routes.
+While running ``net-setup.sh`` requires root access, afterwards Zephyr
+process can be run as a non-root user.
+
+You can run the ``net-setup.sh`` script like this::
+
+   cd net-tools
+   ./net-setup.sh
```

Comment:
```suggestion
   sudo ./net-setup.sh
```

Reply:
There is no need to put sudo here. The script will do sudo automatically if user does not put it there.